build.rs

```diff
@@ -4,11 +4,30 @@ use std::path::PathBuf;
 use cmake::Config;
 
 fn main() {
-    let dst = Config::new("HiGHS")
-        .define("FAST_BUILD", "ON")
-        .define("SHARED", "OFF")
-        .define("CMAKE_MSVC_RUNTIME_LIBRARY", "MultiThreadedDLL")
-        .build();
+    // Targets
+    let target = env::var("TARGET").unwrap();
+    let apple = target.contains("apple");
+    let windows = target.contains("windows");
+    let linux = target.contains("linux");
+    let musl = target.contains("musl");
+
+    let dst = if musl {
+        Config::new("HiGHS")
+            .define("FAST_BUILD", "ON")
+            .define("SHARED", "OFF")
+            .define("CMAKE_MSVC_RUNTIME_LIBRARY", "MultiThreadedDLL")
+            .define(
+                "CMAKE_CXX_STANDARD_LIBRARIES",
+                "-static-libgcc -static-libstdc++",
+            )
```

Comment:
Can you remove all the duplicated code and just keep the additional define for musl ?